hoomd/md/pair/aniso.py

```diff
@@ -328,6 +328,54 @@ class ALJ(AnisotropicPair):
     only ``vertices`` creates a convex polytope (set ``vertices`` and ``faces``
     to empty lists to create the ellipsoid).
 
+    Important:
+        `ALJ` implicitly rounds the given shape by :math:`\sigma_c` via the WCA
+        or Lennard-Jones potential.
+
+    .. rubric:: Choosing `r_cut <hoomd.md.pair.Pair.r_cut>`:
+
+    Set `r_cut <hoomd.md.pair.Pair.r_cut>` for each pair of particle types so
+    that `ALJ` can compute interactions for all possible relative placements
+    and orientations of the particles. The furthest apart two particles can be
+    while still interacting depends on the value of ``alpha`` (where :math:`R_i`
+    is the circumsphere radius of the particle with type :math:`i`:):
+
+    * For alpha=0:
+
+      .. math::
+
+        r_{\mathrm{cut},ij} = \max \left( \frac{2^\frac{1}{6}}{2}
+        (\sigma_i + \sigma_j), R_i + R_j + R_{\mathrm{rounding},i} +
```

Comment:
Rendering locally, I think we need more space after the comma.

I also like how 1/6 takes up less vertical space than \frac{1}{6}.

```suggestion
        r_{\mathrm{cut},ij} = \max \left( \frac{2^{1/6} (\sigma_i + \sigma_j) }{2}
        \: , R_i + R_j + R_{\mathrm{rounding},i} +
```